pkg/api/v1/exposure_types.go

```diff
@@ -0,0 +1,142 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1
+
+import "time"
+
+// The following constants are generally useful in implementations of this API
+// and for clients as well..
+const (
+	// only valid exposure key keyLength
+	KeyLength = 16
+
+	// Transmission risk constraints (inclusive..inclusive)
+	MinTransmissionRisk = 0 // 0 indicates, no/unknown risk.
+	MaxTransmissionRisk = 8
+
+	// Intervals are defined as 10 minute periods, there are 144 of them in a day.
+	// IntervalCount constraints (inclusive..inclusive)
+	MinIntervalCount = 1
+	MaxIntervalCount = 144
+
+	// interval length
+	IntervalLength = 10 * time.Minute
+
+	// Error Code defintiions.
+	// ErrorUnknownHealthAuthorityID indicates that the health authority was not found.
+	ErrorUnknownHealthAuthorityID = "unknown_health_authority_id"
```

Comment:
These aren't really errors - they are strings 😄 . If we want them to be errors, I recommend doing this:

```go
type Error string

func (e Error) Error() string {
  return e
}

const (
  ErrBadThing = Error("bad thing")
)
```